evergreen/run-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+MONGODB_URI=${MONGODB_URI:=mongodb://localhost:27017/}
+
+dotnet clean "./MongoDB.AspNetCore.OData.sln"
+for TEST_PROJECT in ./tests/**/*.Tests.csproj; do
+  ## Evergreen require JunitXml test results format. Have to add logger as the dependency for test project
+  dotnet add "$TEST_PROJECT" package JunitXml.TestLogger
```

Comment:
What's the benefit of adding this dynamically and removing? (As opposed to adding this like we do in driver) ?

Reply:
This let us not to have "unnecessary" dependency, that someone might want to remove as not-in-use. But I'm OK with adding this as dependency (in fact I have not found how it was referenced in Driver solution at that moment, now I see that we have it in Directory.build file - we can do the same thing for oData)